salt/modules/schedule.py

```diff
@@ -88,8 +88,10 @@ def _get_schedule_config_file():
         ),
     )
 
-    if not os.path.isdir(minion_d_dir):
-        os.makedirs(minion_d_dir)
+    if not os.path.isdir(config_dir):
+        os.makedirs(config_dir)
+        if not os.path.isdir(minion_d_dir):
+            os.makedirs(minion_d_dir)
```

Comment:
**question** should this be nested? That is, it will only be created when the config dir doesn't exist... Should the `minion_d_dir` always be created if it doesn't exist?